.circleci/config.yml

```diff
@@ -0,0 +1,69 @@
+version: 2.1
+jobs:
+  new-tag:
+    executor: docker-executor
+    steps:
+      - checkout
+      - commit-next-tag
+  release:
+    docker:
+      - image: circleci/golang:1.13
+    working_directory: /go/src/github.com/astronomer/astro-cli
+    steps:
+      - checkout
+      - run: curl -sL https://git.io/goreleaser | bash
+workflows:
+  version: 2.1
+  build-images:
+    jobs:
+      - approve-release:
+          type: approval
+          filters:
+            branches:
+              only: '/release-.*/'
+      - new-tag:
+          requires:
+            - approve-release
+          filters:
+            branches:
+              only: '/release-.*/'
```

Comment:
Are we not running any tests on PRs/master branch etc?

Reply:
Tests are still in drone I believe. We'll be migrating all the testing phases over next. Initial step was just to move the release process into Circle, which mostly deals with the `release-x` branches.